app/services/rm_api_service.rb

```diff
@@ -64,4 +76,27 @@ def errors
       [{ title: "Unhandled Error" }]
     end
   end
+
+  class Responses
+    def initialize(responses)
+      @responses = responses.values
+      responses.each do |key, res|
+        define_singleton_method(key) { res }
+      end
+    end
+
+    def ok?
+      @responses.all?(&:ok?)
+    end
+
+    def code
+      @responses.last.code
+    end
+
+    def errors
+      @responses.reduce([]) do |memo, res|
```

Comment:
If we do want to keep the break point above in `request_multi`, this shouldn't be necessary since only the very last request will have been not OK. (same goes for the `ok?` method above)